### PR TITLE
Feature/df 1339 show statuses

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/competence-catalog/shared/competence-set/competence-set.component.html [hunk 1]
```diff
@@ -11,6 +14,15 @@
...
       <span class="mx-3">{{ 'portal.competence-catalog.competence-sets.overview.competence-set-panel-header' | translate }}</span>
       <span class="mx-3">{{ competenceSet.id }}</span>
+      <ng-container *ngIf="observables.isCompetenceCatalogEditor && showStatuses">
+        <span>{{isPublished | booleanToText:
+            'portal.competence-catalog.publication-statuses.published':
+            'portal.competence-catalog.publication-statuses.not-published' | translate}}</span>
+        &nbsp;&&nbsp;
+        <span>{{isDraft | booleanToText:
+            'portal.competence-catalog.draft-statuses.draft':
+            'portal.competence-catalog.draft-statuses.not-draft' | translate}}</span>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/shared/competence-set/competence-set.component.html]: rename to portal.competence-catalog.draft-statuses.approved

File: alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item/interactive-list-item.component.html [hunk 2]
```diff
@@ -8,9 +11,20 @@
...
-      </small>
+        <ng-container *ngIf="observables.isCompetenceCatalogEditor && showStatuses">
+            -
+            <span>{{isPublished | booleanToText:
+                'portal.competence-catalog.publication-statuses.published':
+                'portal.competence-catalog.publication-statuses.not-published' | translate}}</span>
+            &
+            <span>{{isDraft | booleanToText:
+                'portal.competence-catalog.draft-statuses.draft':
+                'portal.competence-catalog.draft-statuses.not-draft' | translate}}</span>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item/interactive-list-item.component.html]: rename to portal.competence-catalog.draft-statuses.approved

File: alv-portal-ui/src/app/competence-catalog/shared/competence-set/competence-set.component.html (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/shared/competence-set/competence-set.component.html]: done

File: alv-portal-ui/translations.csv [hunk 3]
```diff
@@ -1168,6 +1168,10 @@ portal.competence-catalog.sorting.DATE_DESC,Erstellungsdatum absteigend,Creation
 portal.competence-catalog.sorting.DATE_ASC,Erstellungsdatum aufsteigend,Creation date ascending,Date de création croissant,Data di creazione crescente,
 portal.competence-catalog.sorting.ALPHA_DESC,Alphabetisch absteigend,Alphabetically descending,Ordre alphabétique descendant,In ordine alfabetico decrescente,
 portal.competence-catalog.sorting.ALPHA_ASC,Alphabetisch aufsteigend,Alphabetically ascending,Ordre alphabétique croissant,In ordine alfabetico ascendente,
+portal.competence-catalog.draft-statuses.draft,In Bearbeitung,,,,
```
Comment (emina-str) [inline alv-portal-ui/translations.csv]: please check the translations from https://alv-ch.atlassian.net/browse/DF-1667
there is a translation for "published/not published", so only draft/approved translation is missing.

File: alv-portal-ui/translations.csv [hunk 4]
```diff
@@ -1168,6 +1168,10 @@ portal.competence-catalog.sorting.DATE_DESC,Erstellungsdatum absteigend,Creation
 portal.competence-catalog.sorting.DATE_ASC,Erstellungsdatum aufsteigend,Creation date ascending,Date de création croissant,Data di creazione crescente,
 portal.competence-catalog.sorting.ALPHA_DESC,Alphabetisch absteigend,Alphabetically descending,Ordre alphabétique descendant,In ordine alfabetico decrescente,
 portal.competence-catalog.sorting.ALPHA_ASC,Alphabetisch aufsteigend,Alphabetically ascending,Ordre alphabétique croissant,In ordine alfabetico ascendente,
+portal.competence-catalog.draft-statuses.draft,In Bearbeitung,In progress,En élaboration,In lavorazione,
+portal.competence-catalog.draft-statuses.approved,Freigegeben,Released,Validé,Approvato,
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: Released? this should be approved

**Pull request merged**